src/core/document.js

```diff
@@ -641,56 +641,64 @@ class Page {
   }
 
   get _parsedAnnotations() {
-    const parsedAnnotations = this.pdfManager
-      .ensure(this, "annotations")
-      .then(() => {
-        const annotationPromises = [];
-        for (const annotationRef of this.annotations) {
-          annotationPromises.push(
-            AnnotationFactory.create(
-              this.xref,
-              annotationRef,
-              this.pdfManager,
-              this._localIdFactory,
-              /* collectFields */ false
-            ).catch(function (reason) {
-              warn(`_parsedAnnotations: "${reason}".`);
-              return null;
-            })
-          );
+    const acroFormPromise = this.pdfManager.ensureCatalog("acroForm");
```

Comment:
With the comment above addressed, do we actually need to make any changes in this file?